annlite/index.py

```diff
@@ -182,6 +184,9 @@ def __init__(
                     break
             logger.info(f'Total training data size: {total_size}')
 
+        if self.total_docs > 0:
+            self.restore()
```

Comment:
I think this update has been included in previous PR.

Reply:
will rebase